react/features/base/util/parseURLParams.js

```diff
@@ -17,7 +17,7 @@ export function parseURLParams(
         url: URL,
         dontParse: boolean = false,
         source: string = 'hash'): Object {
-    const paramStr = source === 'search' ? url.search : url.hash;
+    const paramStr = url && (source === 'search' ? url.search : url.hash);
```

Comment:
how come this is called without an actual URL?

Reply:
`function getDevicesFromURL(state: Object) {
    const urlParams
        = parseURLParams(state['features/base/connection'].locationURL);`

This is called when configuring the initial devices. Since there's no connection for this case, there's also no URL

Reply:
Then can we perhaps use a different function, since there is no URL really, so we probably shouldn't be trying to load things from there.

Reply:
removed the parent call completely since it turns out it wasn't required